conda_smithy/lint_recipe.py

```diff
@@ -1006,6 +1393,7 @@ def run_conda_forge_specific(meta, recipe_dir, lints, hints):
 
     # 5: Package-specific hints
     # (e.g. do not depend on matplotlib, only matplotlib-base)
+    # TODO: do the same for if selectors
```

Comment:
should we take care of this? Is this about flattening the selectors?

Reply:
I don't understand this comment

Reply:
I've removed this comment. 
It was meant for `rattler-build-conda-compat package` to handle the situation when some of the packages are listed under `then/else ` branches. 
It is done here: https://github.com/nichmor/rattler-build-conda-compat/blob/main/src/rattler_build_conda_compat/loader.py#L90